libraries/aws_cloudfront_distribution.rb

```diff
@@ -68,6 +68,15 @@ def initialize(opts = {})
       end
     end
     @custom_origin_ssl_protocols = @custom_origin_ssl_protocols.uniq.sort
+
+    @s3_origin_configs = false
+    config.origins.items.each do |origin|
+      if origin[:s3_origin_config]
+        @s3_origin_configs = true
+      elsif origin[:custom_origin_config]
+        @custom_origin_protocol_policy = origin[:custom_origin_config][:origin_protocol_policy]
+      end
+    end
```

Comment:
part of the conditional is a duplicate of https://github.com/inspec/inspec-aws/blob/e49f3d686c18d2aed8b871e326ab2029133da0da/libraries/aws_cloudfront_distribution.rb#L63

Reply:
Should I just merge both those attributes into one ? Like this:

```
@custom_origin_ssl_protocols = []
    @s3_origin_configs = false
    config.origins.items.each do |origin|
      if origin[:s3_origin_config]
        @s3_origin_configs = true
      elsif origin[:custom_origin_config]
        @custom_origin_ssl_protocols += origin[:custom_origin_config][:origin_ssl_protocols][:items]
        @custom_origin_protocol_policy = origin[:custom_origin_config][:origin_protocol_policy]
      end
    end
    @custom_origin_ssl_protocols = @custom_origin_ssl_protocols.uniq.sort
```

Reply:
yeah that sounds good ..

Reply:
btw, in the proposed logic, `@custom_origin_protocol_policy` will always be the last item's origin_protocol_policy.. 
We should allow to capture all protocol's that were applied for certain origin rule..

Reply:
fixed this